lxd/device/gpu.go

```diff
@@ -490,9 +490,9 @@ func (d *gpu) deviceLoadGpu(all bool) ([]gpuDevice, []nvidiaGpuDevice, error) {
 						}
 
 						tmpGpu.nvidia.path = nvidiaPath
-						tmpGpu.nvidia.major = shared.Major(stat.Rdev)
```

Comment:
Should we just drop those two functions out of shared then (and convert any remaining caller)?

Reply:
Nevermind, you're doing exactly that in the next commit :)